pinot-common/src/main/java/org/apache/pinot/common/exception/QueryException.java

```diff
@@ -53,6 +53,7 @@ public static void setMaxLinesOfStackTrace(int maxLinesOfStackTracePerFrame) {
   public static final int ACCESS_DENIED_ERROR_CODE = 180;
   public static final int TABLE_DOES_NOT_EXIST_ERROR_CODE = 190;
   public static final int QUERY_EXECUTION_ERROR_CODE = 200;
+  public static final int QUERY_CANCELLATION_ERROR_CODE = 210;
```

Comment:
Let's put `205` because here as `210` is already taken

Reply:
oh, good catch.